docs/examples/airship-battle.mdx

```diff
@@ -157,19 +186,60 @@ This specifies what material the core is made of, who each core belongs to and h
 ```
 
 [<i className="icon-right">➡️</i>](/docs/modules/gear/repair-remove-keep)
-Remove shears, obsidian and gold block items from the playing field when they are dropped.
+We do not want an excessive amount of items dropped on the map, so this allows us to manage how to deal with dropped items more easily.
 
 ```xml
+<!-- These items will drop, players that already have these items can pick it up and will be merged -->
+<toolrepair>
+    <tool>diamond sword</tool>
+    <tool>bow</tool>
+    <tool>diamond pickaxe</tool>
+    <tool>diamond axe</tool>
+</toolrepair>
+<!-- All of these items will be automatically removed when dropped -->
 <itemremove>
-    <item>shears</item>
+    <item>gold helmet</item>
+    <item>chainmail chestplate</item>
+    <item>leather leggings</item>
+    <item>leather boots</item>
     <item>obsidian</item>
+    <item>arrow</item>
+    <item>wood</item>
+    <item>cooked fish</item>
+    <item>ladder</item>
     <item>gold block</item>
+    <item>bucket</item>
+    <item>water bucket</item>
+	<item>golden apple</item>
 </itemremove>
 ```
 
+[<i className="icon-right">➡️</i>](/docs/modules/gear/kill-rewards)
+Reward players by giving various items for killing an enemy.
+
+```xml
+<kill-rewards>
+    <kill-reward>
+        <item amount="16" material="arrow"/>
+        <item damage="1" amount="32" material="wood"/>
+        <item material="golden apple"/>
+    </kill-reward>
+</kill-rewards>
+```
+
+[<i className="icon-right">➡️</i>](/docs/modules/gear/crafting)
+Stop players from being able to craft specific blocks or items.
+
+```xml
+<crafting>
+    <disable>chest</disable>
+    <disable>boat</disable>
+</crafting>
+```
+
 [<i className="icon-right">➡️</i>](/docs/modules/environment/world/#build-height)
 
-This specifies how high players can build however it doesn't stop them from going over this limit.
+This specifies how high players can build, however, it doesn't stop them from going over this limit.
```

Comment:
maybe clarify this applies only to block-place, they can still go, or, break blocks above this limit

Reply:
Would this be good? I can make sure it is updated on other maps with a build height.

```suggestion
This specifies how high players can build, however, it does not stop them from walking or breaking blocks above this limit.
```